demos/CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ else()
     set(COMPILER_IS_GCC_LIKE FALSE)
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm.*|ARM.*)" AND NOT CMAKE_CROSSCOMPILING)
```

Comment:
CMAKE_SYSTEM_PROCESSOR -> CMAKE_HOST_SYSTEM_PROCESSOR ?

Reply:
The [documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_PROCESSOR.html) says that `CMAKE_SYSTEM_PROCESSOR` has the same value as the `CMAKE_HOST_SYSTEM_PROCESSOR` when not cross-compiling.
However, probably, `CMAKE_HOST_SYSTEM_PROCESSOR` usage is more robust.

Reply:
@ilya-lavrenov do you prefer to use `CMAKE_HOST_SYSTEM_PROCESSOR `?

Reply:
both are workable, but `CMAKE_HOST_SYSTEM_PROCESSOR` just emphasizes the fact we are building on host :)